resources/functionMap.php

```diff
@@ -11535,7 +11535,7 @@
 'sqlsrv_prepare' => ['resource|false', 'conn'=>'resource', 'sql'=>'string', 'params='=>'array', 'options='=>'array'],
 'sqlsrv_query' => ['resource|false', 'conn'=>'resource', 'sql'=>'string', 'params='=>'array', 'options='=>'array'],
 'sqlsrv_rollback' => ['bool', 'conn'=>'resource'],
-'sqlsrv_rows_affected' => ['int|false', 'stmt'=>'resource'],
+'sqlsrv_rows_affected' => ['int<-1,max>|false', 'stmt'=>'resource'],
```

Comment:
I would assume this function will also return a `numeric-string` when the number is big enough.
it should have the same technical limitations then other `*row_affected` functions (even if it is not documented in the php-src manual)

Reply:
Even PDO does not document this, only MySQLi, and I suspect it's because it's a deeply PHP-integrated driver, so I think I'd rather leave these as is as I do not know.

Reply:
Thanks for the hint. Lets see, maybe https://github.com/php/doc-en/issues/1483 Is helpfull